src/llmq/utils.cpp

```diff
@@ -579,10 +579,20 @@ std::set<uint256> CLLMQUtils::GetQuorumRelayMembers(const Consensus::LLMQParams&
     std::set<uint256> result;
 
     auto calcOutbound = [&](size_t i, const uint256& proTxHash) {
+        // Handle edge cases
+        std::set<uint256> r;
+        if (mns.size() == 1) {
+            return r;
+        }
+        if (mns.size() == 2) {
+            if (mns[0]->proTxHash == proTxHash) {
+                r.emplace(mns[1]->proTxHash);
+            }
+            return r;
+        }
```

Comment:
Please add documentation for how these are edge cases and why these items are correct